Dockerfile

```diff
@@ -6,9 +6,12 @@ RUN go mod download
 ADD . .
 RUN make build
 
-FROM alpine:latest
+FROM alpine:3.18.3
 WORKDIR /fabconnect
-COPY --from=fabconnect-builder /fabconnect/fabconnect ./
+COPY --from=fabconnect-builder --chown=1001:0 /fabconnect/fabconnect ./
```

Comment:
we might not need this one here bc of the permissions of `/usr/bin` are opened up to other linux users normally